rts/lra/client/src/main/java/io/narayana/lra/client/internal/NarayanaLRAClient.java

```diff
@@ -737,54 +738,53 @@ public URI enlistCompensator(URI uri, Long timelimit, String linkHeader, StringB
 
                 headers.add("Link", linkHeader);
                 response = client.target(coordinatorUrl)
-                    .path(LRAConstants.getLRAUid(uri))
-                    .queryParam(TIMELIMIT_PARAM_NAME, timelimit)
-                    .request()
-                    .headers(headers)
-                    .async()
-                    .put(Entity.text(compensatorData == null ? linkHeader : data))
-                    .get(JOIN_TIMEOUT, TimeUnit.SECONDS);
+                        .path(LRAConstants.getLRAUid(uri))
```

Comment:
These next 25 lines or so move them to the right by a tab - I did this because the control flow alignment was difficult to parse.